geomesa-accumulo/geomesa-accumulo-datastore/src/main/scala/org/locationtech/geomesa/accumulo/data/stats/StatsRunner.scala

```diff
@@ -9,10 +9,11 @@
 package org.locationtech.geomesa.accumulo.data.stats
 
 import java.io.Closeable
+import java.time.temporal.ChronoUnit
+import java.time.{Clock, Instant}
```

Comment:
looks fine, but wondering if there was a reason to switch to java time instead of joda?